web/styles/rendered_markdown.css

```diff
@@ -210,8 +210,15 @@
         }
     }
 
+    a .alert-word {
+        background-color: var(--color-background-alert-word);
+        color: hsl(210deg 100% 25%);
```

Comment:
Let's make a variable for the `color` values too. Also, I don't understand how this doesn't change light mode? Your screenshots suggest that light mode isn't affected.

Reply:
@karlstolley Thank you for highlighting that. I've resolved the issue by making the required adjustments. The color modifications are now specifically tailored to dark mode, and I have included screenshots to confirm that light mode remains unchanged.